hedera-node/hedera-app/src/main/java/com/hedera/node/app/info/DiskStartupNetworks.java

```diff
@@ -212,23 +217,31 @@ public static void tryToExport(@NonNull final Network network, @NonNull final Pa
      * Converts a {@link AddressBook} to a {@link Network}. The resulting network will have no TSS
      * keys of any kind.
      *
-     * @param addressBook the address book to convert
+     * @param addressBook   the address book to convert
+     * @param configuration the configuration
      * @return the converted network
      */
-    public static @NonNull Network fromLegacyAddressBook(@NonNull final AddressBook addressBook) {
+    public static @NonNull Network fromLegacyAddressBook(
+            @NonNull final AddressBook addressBook, @NonNull final VersionedConfiguration configuration) {
         final var roster = buildRoster(addressBook);
+        final var hederaConfig = configuration.getConfigData(HederaConfig.class);
+        final var ledgerConfig = configuration.getConfigData(LedgerConfig.class);
```

Comment:
Reason for extra file changes compared to the PR on `main` is because the configuration is needed here to get `LedgerConfig`